### DISCUSSION TITLE
ImageMagick ONLINE: No Images Will Not Download At All: JPG, JPEG, PNG, nothing FILE NOT FOUND

### DISCUSSION THREAD

**Discussion by ArtTrue** [ImageMagick/ImageMagick#6888] [Help] [answered]
### ImageMagick version

ONLINE version

### Operating system, version and so on

windows 10

### Description

Hello Support,
When work on image is complete, selecting Single-File Output, then clicking the DOWNLOAD button produces FILE NOT FOUND
The same happens when clicking directly on the image for download:  FILE NOT FOUND
Please note that I have been a user of ImageMagick ONLINE since January 5, 2012. That's over a decade!
Below is the URL that is generated for the FILE NOT FOUND when the DOWNLOAD button is clicked:
Any assistance would be greatly appreciated.
ArtTrue

FILE NOT FOUND:
https://magickstudio.imagemagick.org/workarea/ba924363801f17353e84b948706b6e21e0e6fae185e3f7b72990d281d2cb126782a7c3a85118a5724f54cbdf8adba8cd344122f26f3811b7f0191b7e5f627bfa//var/www/html/MagickStudio/workarea/c7e348093c3a974e45eb78b94fe44ce1e9c4d3cbfc3436a5722f09019c7e4b903b31e6976ee63af12093f4592404a0b5c2288eb24204b3b0c557b0851a087bdf/MagickStudio.mpc

**Comment by urban-warrior** [comment]
The issue should be fixed now.

**Reply by ArtTrue** [reply, marked as answer]
Thank You! .... urban-warrior. IM is working again smooth as silk. ALL SYSTEMS GO!
